fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -497,7 +495,9 @@ protected void grantPrivileges(String groupName) {
         final IDatabaseAdapter adapter = getDbAdapter(dbType, connectionPool);
         try (ITransaction tx = TransactionFactory.openTransaction(connectionPool)) {
             try {
-                pdm.applyGrants(adapter, groupName, grantTo);
+                pdm.applyGrants(adapter, FhirSchemaConstants.FHIR_USER_GRANT_GROUP, grantTo);
+                pdm.applyGrants(adapter, FhirSchemaConstants.FHIR_OAUTH_GRANT_GROUP, grantTo);
+                pdm.applyGrants(adapter, FhirSchemaConstants.FHIR_BATCH_GRANT_GROUP, grantTo);
```

Comment:
so, all the db objects will be processed 3 times - once for each group?  even thought maybe only some of them belong to certain group ...

Reply:
well, if I understand it right, `buildCommonModel` will only add the "target" objects to the pdm, so its not necessarilly "all" the db objects.

Reply:
it's not necessarily all, but yes, it'll potentially process three times, applying only where the group matches